ports/windows/mpconfigport.h

```diff
@@ -90,7 +90,7 @@
 #define MICROPY_PY_SYS_ATEXIT       (1)
 #define MICROPY_PY_SYS_PLATFORM     "win32"
 #ifndef MICROPY_PY_SYS_PATH_DEFAULT
-#define MICROPY_PY_SYS_PATH_DEFAULT "~/.micropython/lib"
+#define MICROPY_PY_SYS_PATH_DEFAULT ".frozen;~/.micropython/lib"
```

Comment:
@stinos is `;` the correct separator here for all cases where the windows port will run?  Even if it runs under mingw does it still use `;`?

Reply:
Yes, unix/main.c has `PATHLIST_SEP_CHAR` as `;` for all windows ports, and it's the only place where `MICROPY_PY_SYS_PATH_DEFAULT` is used

Reply:
Great, thanks for confirming.